test/canvas.test.js

```diff
@@ -486,6 +486,24 @@ describe('Canvas', function () {
     assert.equal(ctx.font, '15px Arial, sans-serif')
   })
 
+  it('Context2d#lang=', function () {
+    const canvas1 = createCanvas(200, 200)
+    const ctx1 = canvas1.getContext('2d')
+    assert.equal(ctx1.lang, '')
+    ctx1.font = '20px sans-serif';
+    ctx1.lang = 'ja'
+    assert.equal(ctx1.lang, 'ja')
+    ctx1.fillText('门', 10, 10)
+
+    const canvas2 = createCanvas(200, 200)
+    const ctx2 = canvas2.getContext('2d')
+    ctx2.font = '20px sans-serif';
+    ctx2.lang = 'zh'
+    ctx2.fillText('门', 10, 10)
+
+    assert.notDeepEqual(ctx1.getImageData(10, 10, 40, 40), ctx2.getImageData(10, 10, 40, 40))
+  })
```

Comment:
This test won't work on an OS that doesn't come preinstalled with fonts that support the different languages. I'd be happy with just an update to the `describe('Context2d#save()/restore()'` test. But we could also add a font to the repo if it's small enough.

Reply:
@chearon oh cool, it is working with Source Han Sans! Thanks for finding that. As long as it's documented / searchable here I'm OK with moving forward with the simpler test